node/lib/util/commit.js

```diff
@@ -597,14 +612,22 @@ exports.commit = co.wrap(function *(metaRepo,
 
     const tree = yield index.writeTree();
     const headCommit = yield metaRepo.getHeadCommit();
+    const parents = [headCommit];
+    if (mergeParent) {
+        parents.push(mergeParent);
```

Comment:
what happens if `mergeParent.id() === headCommit.id()`? Will it be two parents of the same commit?

Reply:
I think git meta merge prevents you from starting a merge like that, but I can add an assert.